### PR TITLE
Internal Formatting Changes, Bedrock Miner Docs

### PR DESCRIPTION
**Pull request by mikerooni**
- Adds docs for bedrock miners
- Reformatted long lines and made a few other minor formatting changes

### CODE REVIEW

File: docs/Modpacks/Other-Topics/Adding-and-Removing-Recipes.md [hunk 1]
```diff
@@ -66,9 +69,18 @@ ServerEvents.recipes(event => {
...
-    - `.output()`: As above, but defines an output instead. Takes the exact same parameters. This method is likewise unwieldy to use; it is more user-friendly to use the ones below that clearly tell you what output type they call.
+    - `.input()`:  
+      The most basic input definition available. Takes two parameters: one RecipeCapability that defines what input type
+      this call is supposed to be (usually an item, a fluid or energy, but can also be, for example, Create Stress
+      Units), and an Object that defines the input itself. Available RecipeCapabilities can be found in the GTCEu Modern
```
Comment (screret) [inline docs/Modpacks/Other-Topics/Adding-and-Removing-Recipes.md]: SU isn't valid anymore.

File: docs/Modpacks/Other-Topics/Adding-and-Removing-Recipes.md [hunk 2]
```diff
@@ -110,7 +133,9 @@ ServerEvents.recipes(event => {
 })
 ```
 
-1. Note due to a `JS Moment` you have to force `scannerResearch` to be interpreted by Rhino in a specific way. Scanner Research accepts an `ItemStack` input in the `.researchStack()` object, and you can also define the `EUt` and `Duration` outside of the `.researchStack()` object.
+1. Note due to a `JS Moment` you have to force `scannerResearch` to be interpreted by Rhino in a specific way.
```
Comment (screret) [inline docs/Modpacks/Other-Topics/Adding-and-Removing-Recipes.md]: pls make JS Moment not be a code block it's so bad

Comment (mikerooni) [reply docs/Modpacks/Other-Topics/Adding-and-Removing-Recipes.md]: I'm just gonna reword it entirely

File: docs/Modpacks/Materials-and-Elements/Material-Properties.md [hunk 3]
```diff
@@ -5,41 +5,54 @@ title: Material Properties
...
-
-4. Overrides the EBF's default behaviour for EU/t.
-
-5. Used for creating a dust material. The haverst level and burn time can be specified in the brackets. Example: `.dust(2, 4000)`
-
-6. Creates a fluid pipe out of the material it is added to. The possible values are: Max Fluid Temperature, Throughput, Gas Proof, Acid Proof, Cyro Proof, Plasma Proof,
+## Blast Furnace Properties
+
+### `BlastProperty.blastTemp()`
+Sets the Blast Furnace Temperature of the material. If the temperature is below 1000K recipes will be generated in the Primitive Blast Furnace. If above 1750K recipes for the Hot Ingot will be created along with the Vacuum Freezer Recipe to cool the ingot. Example: `.blastTemp(2750)`
```
Comment (JuiceyBeans) [inline docs/Modpacks/Materials-and-Elements/Material-Properties.md:11]: > If above 1750K recipes

Probably reword it to "If temperature is above 1750K, recipes" or "If above 1750K, recipes" (add a comma) for better readability

File: docs/Modpacks/Materials-and-Elements/Material-Properties.md [hunk 4]
```diff
@@ -5,41 +5,54 @@ title: Material Properties
...
+
+### `BlastProperty.gasTier()`
+Sets the Gas Tier which determins what GAS EBF recipes will be generated. Example: `.gasTier(LOW)`
```
Comment (JuiceyBeans) [inline docs/Modpacks/Materials-and-Elements/Material-Properties.md:14]: > what GAS EBF

Is GAS supposed to be in all caps?

File: docs/Modpacks/Materials-and-Elements/Material-Properties.md [hunk 5]
```diff
@@ -5,41 +5,54 @@ title: Material Properties
...
+### `BlastProperty.eutOverride()`
+Overrides the EBF's default behaviour for EU/t.
+
+
+## `DustProperty.dust()`
+Used for creating a dust material. The haverst level and burn time can be specified in the brackets. Example: `.dust(2, 4000)`
+
+
+## `FluidPipeProperty.fluidPipeProperties()`
+Creates a fluid pipe out of the material it is added to. The possible values are: Max Fluid Temperature, Throughput, Gas Proof, Acid Proof, Cyro Proof, Plasma Proof,
```
Comment (JuiceyBeans) [inline docs/Modpacks/Materials-and-Elements/Material-Properties.md:28]: > Cyro Proof

*Cryo Proof

File: docs/Modpacks/Materials-and-Elements/TagPrefixes-and-the-power-of-.setIgnored().md [hunk 6]
```diff
@@ -1,46 +1,78 @@
...
-which associates with all Materials that have an IngotProperty and thus an associated ingot item, including custom ones defined via KubeJS. 
-TagPrefixes provide localization, item and block tagging, and influence many crafting recipes, and are integral to the functioning of GTCEu's material definition system.
+
+TagPrefixes are GTCEu Modern's way of streamlining applying item and block tags to Materials, along with some other
+functions. The `TagPrefix` class, available for use in startup and server scripts, contains a number of predefined
+TagPrefixes that associate potentially everything from drill heads to flawless gemstones with a material.
+
+A common and easy-to-understand example is `TagPrefix.ingot`, which associates with all Materials that have an
+IngotProperty and thus an associated ingot item, including custom ones defined via KubeJS.  
+TagPrefixes provide localization, item and block tagging, and influence many crafting recipes, and are integral to the
```
Comment (JuiceyBeans) [inline docs/Modpacks/Materials-and-Elements/TagPrefixes-and-the-power-of-.setIgnored().md:16]: > tagging, and influence many crafting recipes, and are integral

Redundant and at "and influence", should probably be "[...] tagging, influence many crafting recipes, and are integral [...]"

File: docs/Modpacks/Materials-and-Elements/The-ChemicalHelper-class.md [hunk 7]
```diff
@@ -1,42 +1,76 @@
...
-title: The ChemicalHelper class
+title: ChemicalHelper
 ---
 
-It may behoove a packmaker working with GTCEu Modern to learn about the ChemicalHelper class. This class, available for use in server scripts, 
-contains a number of useful methods that can ease working with GTCEu Materials in contexts where it might not be possible, or it might be unsafe, to work with item or block tags.
+
+# The `ChemicalHelper` Class
+
+It may behoove a packmaker working with GTCEu Modern to learn about the ChemicalHelper class.
```
Comment (JuiceyBeans) [inline docs/Modpacks/Materials-and-Elements/The-ChemicalHelper-class.md:8]: > behoove

Listen, I love my lesser known words, but even I had to google what behoove meant :trolllaugh:
Probably just say its useful instead lol

File: docs/Modpacks/Other-Topics/Adding-and-Removing-Recipes.md [hunk 8]
```diff
@@ -7,7 +7,8 @@ title: "Adding & Removing Recipes"
 
 ## Removing Recipes
 
-Removing GTCEu Modern recipes with KubeJS works the same as any other recipe, meaning they can be removed by: ID, Mod, Input, Output, Type or a Mixture.
+Removing GTCEu Modern recipes with KubeJS works the same as any other recipe, meaning they can be removed by: 
+ID, Mod, Input, Output, Type or a Mixture.
```
Comment (JuiceyBeans) [inline docs/Modpacks/Other-Topics/Adding-and-Removing-Recipes.md:11]: Probably decapitalise Mixture, it seems to imply that Mixture itself is a parameter instead of being a mixture of removal parameters

File: docs/Modpacks/Other-Topics/Adding-and-Removing-Recipes.md [hunk 9]
```diff
@@ -66,9 +69,18 @@ ServerEvents.recipes(event => {
...
-    - `.output()`: As above, but defines an output instead. Takes the exact same parameters. This method is likewise unwieldy to use; it is more user-friendly to use the ones below that clearly tell you what output type they call.
+    - `.input()`:  
+      The most basic input definition available. Takes two parameters: one RecipeCapability that defines what input type
+      this call is supposed to be (usually an item, a fluid or energy), and an Object that defines the input itself.
+      Available RecipeCapabilities can be found in the GTCEu Modern
+      GitHub or the mod's .JAR file, but the class containing all of GTCEu Modern's native RecipeCapabilities,
```
Comment (JuiceyBeans) [inline docs/Modpacks/Other-Topics/Adding-and-Removing-Recipes.md]: Noticed that references to looking at jar files was removed earlier, there's still one here in case you missed it

File: docs/Modpacks/Other-Topics/Adding-and-Removing-Recipes.md [hunk 10]
```diff
@@ -66,9 +69,18 @@ ServerEvents.recipes(event => {
...
+      `GTRecipeCapabilites`, must be manually loaded in your scripts. This method is unwieldy to use in Javascript; it
```
Comment (JuiceyBeans) [inline docs/Modpacks/Other-Topics/Adding-and-Removing-Recipes.md]: *JavaScript
Totally just nitpicking here :trollface:

File: docs/Modpacks/Other-Topics/Adding-and-Removing-Recipes.md [hunk 11]
```diff
@@ -90,14 +102,25 @@ ServerEvents.recipes(event => {
...
+      Using this will enable you to control whether a recipe input/output is consumed/produced per tick the recipe is
+      running or all at once at recipe start/end. Set to true with `.perTick(true)` to make the recipe builder consider
+      any following input/output calls as per-tick. Remember to set the value to false with `.perTick(false)` after the
+      calls you intend to be per-tick, to prevent behaviour you don't want!
+
 
 ### The Research System
-GTCEu has Research System which allows for adding extra requirements to recipes such as: Scanner Research, Station Research and Computation.
+
+GTCEu has Research System which allows for adding extra requirements to recipes such as:
```
Comment (JuiceyBeans) [inline docs/Modpacks/Other-Topics/Adding-and-Removing-Recipes.md:122]: Since its modern docs, probably call it GTCEu Modern instead of just GTCEu here? I noticed its called that in earlier instances too
